docs/OSUpgrades.md

```diff
@@ -169,23 +167,23 @@ container for example.
 From there, probably the simplest thing is to use `oc image extract`
 to unpack the container image.  Something like this:
 ```
-$ mkdir machine-os-content
-$ oc image extract quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:02d810d3eb284e684bd20d342af3a800e955cccf0bb55e23ee0b434956221bdd --path /:machine-os-content
-$ find machine-os-content/srv/repo/ -name '*.commit'
-machine-os-content/srv/repo/objects/33/dd81479490fbb61a58af8525a71934e7545b9ed72d846d3e32a3f33f6fac9d.commit
-$ ostree --repo=machine-os-content/srv/repo ls 33dd81479490fbb61a58af8525a71934e7545b9ed72d846d3e32a3f33f6fac9d
+$ mkdir rhel-coreos
+$ oc image extract quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:6973e9353f29e678cad79fe768c22cfd6697d8aa30d2aeaa78cceea989925ded --path /:rhel-coreos
+$ find rhel-coreos/ -name '*.commit'
+rhel-coreos/sysroot/ostree/repo/objects/bf/445baa02dbbe3bb4b8a6f216c478c78d2e22f45343879e405a12550ba8664a.commit
+$ ostree --repo=rhel-coreos/sysroot/ostree/repo ls bf445baa02dbbe3bb4b8a6f216c478c78d2e22f45343879e405a12550ba8664a
 d00755 0 0      0 /
-l00777 0 0      0 /bin -> usr/bin
-l00777 0 0      0 /home -> var/home
-l00777 0 0      0 /lib -> usr/lib
-l00777 0 0      0 /lib64 -> usr/lib64
-l00777 0 0      0 /media -> run/media
-l00777 0 0      0 /mnt -> var/mnt
-l00777 0 0      0 /opt -> var/opt
-l00777 0 0      0 /ostree -> sysroot/ostree
-l00777 0 0      0 /root -> var/roothome
-l00777 0 0      0 /sbin -> usr/sbin
-l00777 0 0      0 /srv -> var/srv
+l00777 1000 1000      0 /bin -> usr/bin
```

Comment:
Not a big deal but this looks...wrong, that content shouldn't be owned by uid 1000.